src/app/content/practiceQuestions/components/ShowPracticeQuestions.tsx

```diff
@@ -107,29 +119,11 @@ const ShowPracticeQuestions = () => {
       {section ? <SectionTitle dangerouslySetInnerHTML={{ __html: section.title }} /> : null}
       {questionsCount === 0 && nextSection
         ? <EmptyScreen nextSection={nextSection} />
-        : questionsInProggress
-          ? (
-            <QuestionsWrapper>
-              <QuestionsHeader>
-                <FormattedMessage id='i18n:practice-questions:popup:questions'>
-                  {(msg: string) => msg}
-                </FormattedMessage>
-              </QuestionsHeader>
-              <ProgressBar total={questionsCount} activeIndex={currentQuestionIndex} />
-              <Question />
-            </QuestionsWrapper>
-          )
-          : hasAnswers
-            ? <FinalScreen nextSection={nextSection}/>
-            : <QuestionsWrapper>
-              <QuestionsHeader>
-                <FormattedMessage id='i18n:practice-questions:popup:questions'>
-                  {(msg: string) => msg}
-                </FormattedMessage>
-              </QuestionsHeader>
-              <ProgressBar total={questionsCount} activeIndex={currentQuestionIndex} />
-              <IntroScreen />
-            </QuestionsWrapper>
+        : questionsInProggress ? 
+        practiceQuestionsContent
+        : hasAnswers
+            ? <FinalScreen nextSection={nextSection} />
+            : practiceQuestionsContent
```

Comment:
Is there a way to use `practiceQuestionsContent` only once?

Reply:
Maybe this way:
`: hasAnswers && !questionsInProggress`
`            ? <FinalScreen nextSection={nextSection} />`
`            : practiceQuestionsContent`